docs/reference/one-sdk.mdx

```diff
@@ -15,23 +15,16 @@ OneSDK is a universal API client for Node.js. This page describes OneSDK public
 
 ## Installation
 
-OneSDK is available as Node package under the name `@superfaceai/one-sdk`. For installation in a Node.js run:
+OneSDK is available as Node package under the name `@superfaceai/one-sdk`. For installation in a Node.js environment, run:
 
 ```shell
 npm install @superfaceai/one-sdk
+# Or, if you're using yarn
+yarn add @superfaceai/one-sdk
```

Comment:
Personally I'd keep out the yarn out of any documentation. If you use it, you know how to use it, if you don't, it's annoying. Or if you insist on keeping yarn around, then you should also include pnpm. :wink:

Reply:
Keeping this in line with all our READMEs, any reason to make the docs an outlier in this case?